### ISSUE TITLE
Cursor is default on placeholder text which should be caret.

### ISSUE THREAD

**Issue by ranemihir** [editor-js/header#50]
When you hover the cursor over the **placeholder** text of the header tool, the cursor stays **default**. The expected behaviour is that the cursor should be a **caret**. 
Also to be more specific, the cursor is **caret** for the rest of the Header block, its just the placeholder text which seems to have a problem.

**Comment by nanxiaobei** [comment]
Also when click on the **placeholder** first time, not show **caret**, click again, then show **caret**.

Add CSS below would solve this problem:

```css
.ce-header[contentEditable=true][data-placeholder]::before {
  cursor: text;
  pointer-events: none;
}

**Comment by talyguryn** [comment]
resolved in #50
